<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExhalePressureComfortLevel</class>
 <widget class="QDialog" name="ExhalePressureComfortLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="exhaleComfortTitle">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>16</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#BDBDBD</string>
   </property>
   <property name="text">
    <string>Exhale Pressure Comfort Level</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exhaleComfortCancel">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>170</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exhaleComfortAccept">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>170</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>213</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 49, 49);
border: 1px;
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="level1Radio">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#BDBDBD</string>
       </property>
       <property name="text">
        <string>Level 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="level2Radio">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#BDBDBD</string>
       </property>
       <property name="text">
        <string>Level 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="level3Radio">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#BDBDBD</string>
       </property>
       <property name="text">
        <string>Level 3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>exhaleComfortTitle</zorder>
  <zorder>exhaleComfortCancel</zorder>
  <zorder>exhaleComfortAccept</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
